### PR TITLE
docs: Use sudo and tee uniformly everywhere

### PR DESCRIPTION
**Pull request by Rajpratik71**
Added sudo uniformly everywhere it required as it presence doesn't affect anything , but it help new user's of linux , docker or kubernetes who are not much familiar with internals and also for who doesn't read the documentation line by line and use the guides for command reference and just copy paste for their use.

Also , changed  the way of writing or creating files , now using "tee" because previous way gives permission issue for non sudo users.

New pull replace old #24143

### CODE REVIEW

File: content/en/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -237,14 +237,14 @@ To install version 1.18.3, set `VERSION=1.18:1.18.3`.
 
 Then run
 ```shell
-echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
-echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/$VERSION/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.list
+sudo echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+sudo echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/$VERSION/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.list
```
Comment (sftim) [inline content/en/docs/setup/production-environment/container-runtimes.md:241]: This runs `echo` as root, which is probably not what you intended.

File: content/en/docs/setup/production-environment/container-runtimes.md [hunk 2]
```diff
@@ -269,14 +269,14 @@ To install version 1.18.3, set `VERSION=1.18:1.18.3`.
...
```
Comment (sftim) [inline content/en/docs/setup/production-environment/container-runtimes.md:273]: This runs `echo` as root, which is probably not what you intended.

**Review by sftim — DISMISSED** [review]
I spotted one technical concern.

**Comment by sftim** [comment]
Aside: I don't much like `sudo` myself, but it's the market leader on Unix-like systems, so I can see the argument for documenting it.

File: content/en/docs/setup/production-environment/container-runtimes.md (hunk 1 above)
Comment (Rajpratik71) [reply content/en/docs/setup/production-environment/container-runtimes.md]: moved to common pattern , using tee now

File: content/en/docs/setup/production-environment/container-runtimes.md (hunk 2 above)
Comment (Rajpratik71) [reply content/en/docs/setup/production-environment/container-runtimes.md:277]: moved to common pattern , using **tee** now

**Review by bart0sh — CHANGES_REQUESTED** [review]
lgtm except of one thing. This sentence should be refrased or removed:

> You should execute all the commands in this guide as root. For example, prefix commands with sudo , or become root and run the commands as that user.

As it suggests to add sudo to the commands that would already have sudo if this PR is merged.

**Comment by Rajpratik71** [comment]
> lgtm except of one thing. This sentence should be refrased or removed:
> 
> > You should execute all the commands in this guide as root. For example, prefix commands with sudo , or become root and run the commands as that user.
> 
> As it suggests to add sudo to the commands that would already have sudo if this PR is merged.

removing as it should not be applicable after PR merge.

**Review by bart0sh — APPROVED** [review]
/lgtm

**Pull request merged**